### DISCUSSION TITLE
Zulip

### DISCUSSION THREAD

**Discussion by pdurbin** [bostonopen/bostonopen.github.io#9] [General]
We've been using Quiet (@tryquiet) since the beginning because @holmesworcester is the founder!

Overall, Quiet has been fine, but it's mostly just the two of us talking there. Also, Quiet is not (yet) ideal, in my mind, for open source projects and communities because you have to sign up for Quiet before you can read any messages. As a [SLOPI](http://blog.greptilian.com/2020/01/25/slopi-communication/) guy, I like more transparency. 😄  Here's an issue I opened about this:

- https://github.com/TryQuiet/quiet/issues/1776

I'm piloting Zulip at work for Dataverse and it's working very well: https://dataverse.zulipchat.com

I could maybe create a "bostonopen" channel (stream) there but it seems a bit off-topic for Dataverse.

Or I could ask @szabgab how he feels about a "boston" channel on his Zulip org that's oriented around open source: https://osdc.zulipchat.com . (I mentioned our group in [my intro](https://osdc.zulipchat.com/#narrow/stream/393737-new-users/topic/Philip.20Durbin/near/392707440).)

Or I could even ask the Zulip folks to sponsor an org for us, but they already did for Dataverse and I feel bad asking again (maybe someone else could ask 😅 ).

Anyway, what do you think about using Zulip?

**Comment by szabgab** [comment]
I think having a "boston" channel on the OSDC Zulip instance could be nice.

**Reply by szabgab** [reply]
I've created a [stream called boston](https://osdc.zulipchat.com/#narrow/stream/406743-boston) for you. Feel free to register, play around in the [try-zulip](https://osdc.zulipchat.com/#narrow/stream/393561-try-zulip) stream, introduce yourself in the [new-user](https://osdc.zulipchat.com/#narrow/stream/393737-new-users).

I sort of hope that the OSDC project can become a shared resource of various small open projects and groups that will allow for more interaction among these groups.

**Reply by pdurbin** [reply]
Ha! Thanks! Let me make sure I'm not the only person from Boston who wants to join. I think @HarshKapadia2 is a Zulip fan, at least. I just don't want to me talking to myself. Our Quiet instance is pretty, well, quiet. 😁

**Reply by HarshKapadia2** [reply]
Zulip is nice! I used it for a few months a few years ago! The only issue we faced is that it's not easy to get people to make an account on another chat service and to keep them coming back to interact on the platform even if they do join in. I did join the Boston stream though! See you there!
